WalletWasabi.Tests/UnitTests/WabiSabi/Integration/WabiSabiHttpApiIntegrationTests.cs

```diff
@@ -162,7 +163,7 @@ public async Task MultiClientsCoinJoinTestAsync()
 			var transactionCompleted = new TaskCompletionSource<Transaction>();
 
 			// Total test timeout.
-			using var cts = new CancellationTokenSource(TimeSpan.FromSeconds(30 * expectedInputNumber));
+			using var cts = new CancellationTokenSource(TimeSpan.FromSeconds(20 * expectedInputNumber));
```

Comment:
this is the same factor as connection confirmation, but the expected number of requests can be higher:

1. the expected number of outputs is independent of the number of inputs per participant, and each output registration still requires a range proof, but this still correlates with expectedInputNumber since it's also a product of the number of participants 
2. however, when number of inputs per participant is small, and the ratio of inputs to outputs more or less correlates with the number of reissuance requests, in this case even though one of the factors is small the total duration needs to be longer

i think the correct expression would be something like `20 * participantCount * Math.Max(10, participantCount)`, where 10 should account for reissuance overhead, and ~4-5 outputs per participant with room to spare.